spec.md

```diff
@@ -8,121 +8,160 @@ title: Semantic Specification
+
+These can all be valid timing diagrams for children that "FollowFrom" a parent.
+
+~~~
+    [-Parent Span-]  [-Child Span-]
+
+
+    [-Parent Span--]
+     [-Child Span-]
+
+
+    [-Parent Span-]
+                [-Child Span-]
+~~~
 
 ### Logs
 
-Every Span has zero or more **Logs**, each of which being a timestamped event name, optionally accompanied by a structured data payload of arbitrary size. The event name should be the stable identifier for some notable moment in the lifetime of a Span. For instance, a Span representing a browser page load might add an event for each of the Performance.timing moments.
+Every Span has zero or more **Logs**, each of which being a timestamped event name, optionally accompanied by a structured data payload of arbitrary size. The event name should be the stable identifier for some notable moment in the lifetime of a Span. For instance, a Span representing a browser page load might add an event for each field in [Performance.timing](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceTiming).
 
 While it is not a formal requirement, specific event names should apply to many Span instances: tracing systems can use these event names (and timestamps) to analyze Spans in the aggregate.  For more information, see the [Data Semantics Guidelines](/data-semantics).
 
 ### Tags
 
-Every Span may also have zero or more key/value **Tags**, which do not have timestamps and simply annotate the spans.
+Every Span may also have zero or more key:value **Tags**, which do not have timestamps and simply annotate the spans.
 
-As is the case with Logs, if certain known tag key/values are used for common application scenarios, tracers can choose to pay special attention to them. For more information, see the [Data Semantics Guidelines](/data-semantics).
+As is the case with Logs, if certain known tag key:values are used for common application scenarios, tracers can choose to pay special attention to them. For more information, see the [Data Semantics Guidelines](/data-semantics).
 
-## Inject and Join
+## SpanContext
 
-Spans may be **Injected** into and **Joined** from **Carrier** objects that are used for inter-process communication (e.g., HTTP headers). In this way, Spans may propagate across process boundaries along with sufficient information to join up with the Trace in some remote process.
+Every Span must provide access to a **SpanContext**. The SpanContext represents Span state that must propagate to child Spans and across process boundaries (e.g., a `<trace_id, span_id, sampled>` tuple) and also encapsulates any **Baggage** (see below). SpanContext is used when propagating traces across process boundaries and when creating edges in the trace graph (e.g., ChildOf relationships or other [references](#references)).
```

Comment:
If there was `github.com/english/issues` it would be overflowing with complaints :)

I think this is correct, but it's admittedly a corner case. We are using the word "child" as an adjective here, not as a noun (as it would typically be used, that is), and that changes the rules... Imagine that we only talk about propagating to, say, Spans based on their color; then of course we would write

> "... Span state that must propagate to *red* Spans and across process..."

In this sentence, since "child" is a modifier on Spans, I think (??) what I have is correct. Though note that

> "... Span state that must propagate to child and across process ..."

is (clearly) wrong. *That* would indeed be

> "... Span state that must propagate to children and across process ..."

Anyway, YMMV and I am only like 80% confident about this.